dbusadaptor.cpp

```diff
@@ -22,19 +21,27 @@
 #include <QtQuick/QQuickItem>
 
 DBusAdaptor::DBusAdaptor(QQuickView *view)
-    : QDBusAbstractAdaptor(view), m_view{view}
+    : QDBusAbstractAdaptor(view)
+    , m_view(view)
 {
 }
 
 DBusAdaptor::~DBusAdaptor()
 {
 }
 
-void DBusAdaptor::openFile(const QStringList &files)
+void DBusAdaptor::Open(const QStringList &uris, const QVariantMap &platformData)
```

Comment:
Can be done separately, but pondering would it make sense to offer some common component for this adaptor.